tests/phpunit/api/v3/TaxContributionPageTest.php

```diff
@@ -31,14 +30,20 @@ class api_v3_TaxContributionPageTest extends CiviUnitTestCase {
   protected $halfFinancialAccId;
   protected $halfFinancialTypeId;
 
+  protected $isValidateFinancialsOnPostAssert = FALSE;
```

Comment:
This is the same as the parent - I want to switch to TRUE but there is a bug to track down first - I don't know if it is a test bug or a live bug